src/main/java/com/fryrank/dal/ReviewDALImpl.java

```diff
@@ -19,7 +20,7 @@
 import java.util.List;
 import java.util.Map;
 
-import static com.fryrank.Constants.ACCOUNT_ID_KEY;
+import static com.fryrank.Constants.*;
```

Comment:
A general good Java convention is to not do wildcard imports because then it makes it less clear what we're importing and actually using. I know this is an intellij setting; just Google "Intellij disable wildcard imports" and then you'll see how to disable that feature.